python/ray/experimental/ddp/ddp.py

```diff
@@ -0,0 +1,518 @@
+import logging
+import os
+import time
+import argparse
+from dataclasses import dataclass
+from typing import List, Tuple, Optional
+import torch
+import torch.nn as nn
+import torch.optim as optim
+import torch.distributed as dist
+import torch.multiprocessing as mp
+from torch.nn.parallel import DistributedDataParallel as DDP
+
+import ray
+from ray.air._internal import torch_utils
+from ray.dag import InputNode, MultiOutputNode
+from ray.experimental.collective import allreduce
+
+
+logger = logging.getLogger(__name__)
+
+
+@dataclass
+class Config:
+    """Configuration for the demo DDP model."""
+
+    # Model config.
+    num_layers: int = 2
```

Comment:
You should not have a default value here. Instead, you should specify `default=` in `argparse`.

Reply:
Done.